### ISSUE TITLE
Update landing page

### ISSUE THREAD

**Issue by dadofsambonzuki** [teambtcmap/btcmap.org#48]
To be:

We use OpenStreetMap to properly tag places where you can spend Bitcoin and we display that data in our beautiful apps.

Our apps and the underlying data are free and open source.

Tag it and they will come.

**Comment by dadofsambonzuki** [comment]
New tag line:

"Find bitcoin friendly businesses near you."

**Comment by 3j2009** [comment]
I think we should consider who we are targeting when it comes to copy. Whilst the information about how it works and it being free and open source is important information and should be easily accessible, it's place isn't really on the home page as it's not the main selling point of the product even if it is still key information.

We **should** focus on the benefits, i.e. what this product can do for you as a customer or merchant.
We **shouldn't** focus on its specific features or technology. That information is great to have, but not on the home page.

With the new tagline ""Find bitcoin friendly businesses near you." we have simply and accurately conveyed the benefits for consumers. That's one half of the puzzle solved.

Let's look at the Business owners perspective. They probably want to know:
- How easily can I register? Complexity, speed, knowledge, documentation, 
- How much is this going to cost me?
- Is it safe/private/legal? or in other words as a business owner and potential btcmap registrant, my fears surrounding privacy, law and other such issues should be immediately addressed in order for me to continue.


What this could look like in practice would mean cleaning up the home page in favor of consumers and making the on-boarding process for businesses more intuitive. To achieve this i think we'll need to:

- Home page dedicated to the primary target user: The sat spender. In addition to the new tagline, we've moved the technical text to another info section (this could be About us, Wiki, or somewhere else not on the home page). We've also moved the App icons down the lower end of the page in line with the bottom of the Phone, and lastly we've put a nice big "Open Map" button right under the tagline so people can immediately take it for a test drive and see all that glorious Bitcoin adoption (Most people I've showed are quite surprised to find businesses near them that accept bitcoin.)
- The "add location" splits from the Contribute tab and becomes **Register** , which has a user flow solely dedicated to onboarding our secondary target user: Businesses.
-  Communities tab gets acquired by the "Contribute" tab since adding a community or joining a community is just another way to contribute.
- Support Us tab also gets acquired by the "Contribute" tab since donating is also a form on contribution.

Reason for the tab cleanup is reducing noise, optimizing white space, improving sector categorization and making it easier for users to achieve their task.


![btcmap](https://user-images.githubusercontent.com/126127093/232185072-828e9e05-d5e2-4163-8310-085e7618687a.png)

**Comment by dadofsambonzuki** [comment]
Much cleaner!

1. The open source and open data words can go in about us.
2. I think `Register` should just be `Add Location` for now - merchant onboarding and merchant pages are the next main block of work and we may have some form of registration/verification at that point if we're providing other services. In the meantime it's often non-business owners adding locations.
3. The single wiki link in the nav works and we can create a landing page there for more tech content.
4. `Communities` should be top-level nav I think.  Not just about contributing. Although we will also have `Country` pages soon and perhaps `Continents` and so we need to bear that in mind. Perhaps `County` and `Continents` just go under `Stats` though
5. `Stats` --> `Data`?
6. Nav order should be: `Add location`, `Apps`, `Communities`, `Data`, `Contribute`, `Wiki`.

**Comment by 3j2009** [comment]
> "it's often non-business owners adding locations." 

That's a really interesting point and I'm very curious **why**? Imagine being one of the first few businesses listed in the Yellow Pages business directory, or among the first businesses to show up on Google Maps. That a huge advantage in attracting customers. We need to figure out a way to get businesses to understand that, and self-register. But I guess that's a topic for another day...


1. Agree, the `about us` section seems a more appropriate area for the open source and open data words.
2. I Think the 3 drop-down options `Add Location` `Verify Location` and `Open Tickets` should still remain as dropdowns **until the on-boarding flow is optimized**. but their parent `Contribute` we could perhaps change to `Register`.

My reasoning for `Register` instead of `Contribute` or Add` Location` is

- `Register` is a single word which is more desirable for use in a nav bar than 2 words (such as `Add Location` )
- `Register` feels more familiar and intuitive. 
- `Register` could mean register your business or register a location which is good because it's applicable to both types of registrants, owners or non-owners.

3. Happy to help with the landing page if needed.
4. I see, probably a good call to keep `Communities` in top-level nav for now. Seems like it needs further exploration.
5. Either
6. We need an `Open Map` button in there too, which for web-apps the "launch app" button would typically go in the far right of the corner. 
With the "Open Map" button the most eye catching button in the Nav bar, it would also make sense to place the `Register` link (or whatever text term is decided to be the parent link for that flow) next to the `Open Map` button as that would logically the next most sought after link/button in the nav bar after Open Map. Following the same order of importance principle we would place the next most sought after nav link to the left of `Register` and so on and so forth.

e.g.
 `Stats` `Wiki`  `Communities` `Apps` `Register` `Open Map`

![Home (100vh) 1](https://user-images.githubusercontent.com/126127093/232206580-d4338e55-1cb2-4379-893e-b612f6eadd54.png)

**Comment by dadofsambonzuki** [comment]
1. I like `Register` as a simple, top-level CTA . We can just link to the Add Location page for now, but that will become merchant onboarding flow. There could arguably be a community on-boarding flow, but that's an optimisation for later.
2. Maybe we have a `Locations` top-level nav that kinda mirrors `Communities` and incorporates all the current `Stats` nav, which are all location focussed. 

So, `Wiki`, `Communities`, `Locations`, `Apps`, `Register`.

What do you think?

**Comment by 3j2009** [comment]
I'm keen to hear more!
Is it just a rename of `Stats` to `Locations`?
What would the `Locations` nav link include besides the existing Dashboard, Activity and Leaderboard pages currently in `Stats`?

**Comment by dadofsambonzuki** [comment]
> What would the `Locations` nav link include besides the existing Dashboard, Activity and Leaderboard pages currently in `Stats`?

`Add Location`, `Verify Location` & `Open Tickets`. `Contribute` goes.

'Support Us` remains as top-level nav as is important for sponsorship and funding.

**Comment by 3j2009** [comment]
@dadofsambonzuki just to confirm, is this how you're describing the home page & nav bar? 

- `Register` directs to https://btcmap.org/add-location
- Home page & nav looks like this: [Figma interactive prototype.](https://www.figma.com/proto/vUjpoiSe7cap1PeITcbgwl/BTCMap-(Copy)?page-id=0%3A1&node-id=1948-3705&viewport=15839%2C3464%2C0.62&scaling=min-zoom&starting-point-node-id=1948%3A3705&show-proto-sidebar=1)

P.s. see the color shadow on the "Open Map" button on the home page and let me know what you think. I personally think it looks cool in dark mode, but it might not be everyone's taste.

**Comment by dadofsambonzuki** [comment]
Some small mods:

1. Nav under `Location` should be:

     `Add Location`
     `Verify Location`
     `Open Tickets`
     `Location Dashboard`
     `Tagger Activity`
     `Tagger Leaderboard`


2. Nav under `Communities` should be:

     `Community Directory`
     `Add Community` - https://btcmap.org/communities/add
     `Community Leaderboard`

3. `Wiki` shouldn't have a drop down and should link directly to `https://github.com/teambtcmap/btcmap-general/wiki` We can handle signposting in the wiki home page.

4. There is an argument to have a top-level nav for `Data`:

     `Dashboard`
     `Tagger Activity`
     `Tagger Leaderboard`
     `Community Leaderboard`

     Removing those items from the the other menus. I change my mind on that often (!), but today I think it's cleaner under `Data` and makes `Location` less clutered. Thoughts welcomed.

**Comment by 3j2009** [comment]
Okay I've put those changes into the [Prototype](https://www.figma.com/proto/vUjpoiSe7cap1PeITcbgwl/BTCMap-(Copy)?page-id=0%3A1&node-id=1948-3705&viewport=20184%2C3481%2C0.75&scaling=min-zoom&starting-point-node-id=1948%3A3705)

Since `Add Location` would be under the `Locations` dropdown there's no need for a `Register` link in nav. Is this right?

https://user-images.githubusercontent.com/126127093/233588879-1a2a7037-73c0-4f82-bf23-741e5bbb17df.mp4

**Comment by dadofsambonzuki** [comment]
Yeah, I think we should perhaps leave `Register` for now and introduce it to top level nav when when launch merhcant onboarding flow.

Nav still not quite right:

Nav under Location should be:

Add Location
Verify Location
Open Tickets

Nav under Communities should be:

Community Directory
Add Community - https://btcmap.org/communities/add
Community Leaderboard

Nav under Data should be:

Dashboard
Tagger Activity
Tagger Leaderboard
Community Leaderboard

**Comment by secondl1ght** [comment]
Hey guys I'll get caught up on this thread later, I'm working on some other tasks right now.

**Comment by 3j2009** [comment]
ok. I'm not sure we should put `Community Leaderboard` twice. I would suggest it go in `Communities` so that it can then also be shortened to simply `Leaderboard` and the same for the Tagger Leaderboard located in `Data`. Or even better, we have a single `Leaderboards` tab in `Data` which the user can then easily access all leaderboards.

**Comment by dadofsambonzuki** [comment]
Well be introducing a country directory and country leaderboard soon so we should make nav decisions with that in mind.

**Comment by 3j2009** [comment]
@dadofsambonzuki @secondl1ght please confirm if the nav buttons below are correct.

_(ordered R-L)_

**`Open Map`** (furthest right)

**`Locations`**
Add Location
Verify Location
Open Tickets


**`Communities`**
Community Map
Directory
Leaderboard
Add Community


**`Data`**
Dashboard
Tagger Activity
Leaderboard

**`Apps`**
**`Wiki`**
**`Support Us`** (furthest left)
